lib/mqtt.ex

```diff
@@ -0,0 +1,79 @@
+defmodule Mqtt do
+  @moduledoc """
+  `Oc2Mqtt` is main module for handling openc2 over mqtt
+  """
+
+  require Logger
+
+  @doc """
+  Start initializes system variables
+  and starts supervisor of mqtt client
+  """
+  def start do
+    client_id =
+      System.get_env("CLIENT_ID") ||
+        raise """
+        environment variable CLIENT_ID is missing.
+        For example:
+        export CLIENT_ID=:sfractal2020
+        """
+
+    Logger.info("client_id is #{client_id}")
+
+    mqtt_host =
+      System.get_env("MQTT_HOST") ||
```

Comment:
We can manually have this exported on the instance for now while we figure out whether to use the file or the environment variables